python/ray/serve/_private/autoscaling_state.py

```diff
@@ -577,11 +580,29 @@ def __init__(
         self._deployment_autoscaling_states: Dict[
             DeploymentID, DeploymentAutoscalingState
         ] = {}
+        self._policy: Optional[
+            Callable[
+                [Dict[DeploymentID, AutoscalingContext]],
+                Tuple[Dict[DeploymentID, int], Optional[Dict[str, Dict]]],
+            ]
+        ] = None
+        self._policy_state: Optional[Dict[str, Any]] = None
 
     @property
     def deployments(self):
         return self._deployment_autoscaling_states.keys()
 
+    def register(
+        self,
+        autoscaling_policy: AutoscalingPolicy,
+    ):
+        """Register or update application-level autoscaling config and deployments."""
```

Comment:
add to docstring saying this will override deployment level autoscaling

Reply:
done